api service integration changes (#5958)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/uapiservice/protocol-supported.md

### Integration supported

> This guide covers submissions that use [API service](/docs/guides/oin-api-service-overview/)

Reviewer: ```suggestion
This guide covers submissions for an [API service](/docs/guides/oin-api-service-overview/)
```
